### ISSUE TITLE
there seems to be no "EnumDisplaySettings" function to change Screen options

### ISSUE THREAD

**Issue by insinfo** [halildurmus/win32#661]
I'm trying to create a small win32 application to change the monitor rotation and I can't find this function which is needed to manipulate the monitor

```c++
 DEVMODE dm;
   // initialize the DEVMODE structure
   ZeroMemory(&dm, sizeof(dm));
   dm.dmSize = sizeof(dm);

   if (0 != EnumDisplaySettings(NULL, ENUM_CURRENT_SETTINGS, &dm))
   {
      // swap height and width
      DWORD dwTemp = dm.dmPelsHeight;
      dm.dmPelsHeight= dm.dmPelsWidth;
      dm.dmPelsWidth = dwTemp;

      // determine new orientaion
      switch (dm.dmDisplayOrientation)
      {
      case DMDO_DEFAULT:
         dm.dmDisplayOrientation = DMDO_270;
         break;
      case DMDO_270:
         dm.dmDisplayOrientation = DMDO_180;
         break;
      case DMDO_180:
         dm.dmDisplayOrientation = DMDO_90;
         break;
      case DMDO_90:
         dm.dmDisplayOrientation = DMDO_DEFAULT;
         break;
      default:
         // unknown orientation value
         // add exception handling here
         break;
      }
      long lRet = ChangeDisplaySettings(&dm, 0);
      if (DISP_CHANGE_SUCCESSFUL != lRet)
      {
         // add exception handling here
      }
   }
```
I'm following this tutorial
https://learn.microsoft.com/en-us/previous-versions/ms812499(v=msdn.10)?redirectedfrom=MSDN

**Comment by insinfo** [comment]
I ended up implementing here the necessary functions and structs to rotate and manipulate the displays

https://github.com/insinfo/monitor_config/
https://github.com/insinfo/monitor_config/blob/main/lib/src/native_methods.dart
https://github.com/insinfo/monitor_config/blob/main/lib/src/display_device_struct.dart

**Comment by timsneath** [comment]
Thanks -- these APIs are now available in win32 v4.1.1.
